### PR TITLE
Add Prometheus project to allow index files larger than 64GB.

### PR DESCRIPTION
**Pull request by krasi-georgiev**
ping @brian-brazil

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
The [postings compression project](https://github.com/cncf/soc/commit/61a095b0a8be33b7045b3d76ba8c29e4469793bc#diff-04c6e90faac2675aa89e2176d2eec7d8) and this one are quite related so wondering whether should join both into a single one.

File: README.md [hunk 1]
```diff
@@ -167,6 +167,13 @@ Prometheus ideas:
 -	Mentor(s): Goutham Veeramachaneni (@gouthamve) / Krasi Georgiev (@krasi-georgiev)
 -	Issue: https://github.com/prometheus/tsdb/issues/234
 
+#### Allow index files larger than 64GB.
+
+-	This would require some code refactoring on how the index file is saved to disk using varints or some compression.
+-	Recommended Skills: Golang, Compression algorithms
```
Comment (brian-brazil) [inline README.md]: Compression isn't relevant here, no matter what compression we have there'll always be a limit. We should have no limit.

Comment (krasi-georgiev) [reply README.md]: updated

**Comment by krasi-georgiev** [comment]
> The [postings compression project](https://github.com/cncf/soc/commit/61a095b0a8be33b7045b3d76ba8c29e4469793bc#diff-04c6e90faac2675aa89e2176d2eec7d8) and this one are quite related so wondering whether should join both into a single one.

@brian-brazil what do you think about this comment?

**Comment by brian-brazil** [comment]
They're related but different projects.

**Comment by krasi-georgiev** [comment]
I am just not sure if a single student can apply and work on more than one project and neither of these would take full 3 months hence the idea to join as a single one.

Anyway feel free to merge as is.

**Pull request merged**